.fixtures.yml

```diff
@@ -13,6 +13,7 @@ fixtures:
     foreman:       "git://github.com/theforeman/puppet-foreman"
```

Comment:
I think you may not require all repositories, but I suspect you will still need most of them. Doing a full check can't hurt though :)